tools/Customization/DevHome.Customization/Strings/en-us/Resources.resw

```diff
@@ -1,304 +1,392 @@
+              <xsd:attribute name="type" type="xsd:string" />
+              <xsd:attribute name="mimetype" type="xsd:string" />
+              <xsd:attribute ref="xml:space" />
+            </xsd:complexType>
+          </xsd:element>
+          <xsd:element name="assembly">
+            <xsd:complexType>
+              <xsd:attribute name="alias" type="xsd:string" />
+              <xsd:attribute name="name" type="xsd:string" />
+            </xsd:complexType>
+          </xsd:element>
+          <xsd:element name="data">
+            <xsd:complexType>
+              <xsd:sequence>
+                <xsd:element name="value" type="xsd:string" minOccurs="0" msdata:Ordinal="1" />
+                <xsd:element name="comment" type="xsd:string" minOccurs="0" msdata:Ordinal="2" />
+              </xsd:sequence>
+              <xsd:attribute name="name" type="xsd:string" use="required" msdata:Ordinal="1" />
+              <xsd:attribute name="type" type="xsd:string" msdata:Ordinal="3" />
+              <xsd:attribute name="mimetype" type="xsd:string" msdata:Ordinal="4" />
+              <xsd:attribute ref="xml:space" />
+            </xsd:complexType>
+          </xsd:element>
+          <xsd:element name="resheader">
+            <xsd:complexType>
+              <xsd:sequence>
+                <xsd:element name="value" type="xsd:string" minOccurs="0" msdata:Ordinal="1" />
+              </xsd:sequence>
+              <xsd:attribute name="name" type="xsd:string" use="required" />
+            </xsd:complexType>
+          </xsd:element>
+        </xsd:choice>
+      </xsd:complexType>
+    </xsd:element>
+  </xsd:schema>
+  <resheader name="resmimetype">
+    <value>text/microsoft-resx</value>
+  </resheader>
+  <resheader name="version">
+    <value>2.0</value>
+  </resheader>
+  <resheader name="reader">
+    <value>System.Resources.ResXResourceReader, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <resheader name="writer">
+    <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <data name="BrowseButton.Content" xml:space="preserve">
+    <value>Browse</value>
+    <comment>Browse to choose directory</comment>
+  </data>
+  <data name="ChooseDirectory.PlaceholderText" xml:space="preserve">
+    <value>Choose a directory</value>
+    <comment>Choose directory</comment>
+  </data>
+  <data name="ChooseDirectoryPromptText" xml:space="preserve">
+    <value>Choose directory on Dev Drive...</value>
+    <comment>Choose directory on dev drive</comment>
+  </data>
+  <data name="ChosenDirectoryNotOnDevDriveErrorText" xml:space="preserve">
+    <value>Selected directory must be on a Dev Drive</value>
+    <comment>Chosen directory not on dev drive</comment>
+  </data>
+  <data name="DevDriveFreeSizeText" xml:space="preserve">
+    <value>{0} {1} free</value>

```

Comment:
@bbonaby do you know if this needs to be locked?  I know you dealt with sizes in DevDrive.